lib/retry/delay_streams.ex

```diff
@@ -17,7 +19,7 @@ defmodule Retry.DelayStreams do
       end
 
   """
-  @spec exponential_backoff(pos_integer(), pos_integer()) :: Enumerable.t()
+  @spec exponential_backoff(pos_integer(), factor()) :: Enumerable.t()
   def exponential_backoff(initial_delay \\ 10, factor \\ 2) do
     Stream.unfold(initial_delay, fn last_delay ->
       {last_delay, round(last_delay * factor)}
```

Comment:
If `factor` is actually just a `pos_integer` then we would not need to `round`. Since this `round` is here, people must be using `float` for the factor (I would like to!).